### ISSUE TITLE
Keywords change on different run for same text

### ISSUE THREAD

**Issue by vertika-19** [summanlp/textrank#54]
Everytime you run the program, the set of keywords change. Can you point out why is it happening.

**Comment by fedelopez77** [comment]
There are no random components in the algorithm. Can you provide a concrete example?

**Comment by vertika-19** [comment]
Text: I drew this optical illusion in Adobe Illustrator on September 26, 2016. To create it, I used the effect of Akioshi Kitaoka. This is a white and black stroke on a colored background, which sets in motion the focus of vision and it seems to a person that the details of the image are moving. Japanese psychotherapist Yamamoto Hashima has nothing to do with this picture. Moreover, Yamamoto Hashima does not really exist.

Keyword ( Run 1) : ['yamamoto', 'japanese psychotherapist', 'moving', 'hashima', 'kitaoka']
Keyword (Run 2):  ['akioshi', 'kitaoka', 'black stroke', 'colored', 'background']


I'm using as : 
from summa import keywords
ratio = 0.8
keyphrases = keywords.keywords(text, ratio=ratio, split=True)[:5]

**Comment by vertika-19** [comment]
You can check in this different runs are giving different set of keywords as well as order is also changing.

![capture1](https://user-images.githubusercontent.com/14014346/49860071-08360780-fe1f-11e8-8dcd-8de3ea89f6d5.PNG)
![capture2](https://user-images.githubusercontent.com/14014346/49860072-08360780-fe1f-11e8-8b95-77c4e3baebfc.PNG)

**Comment by fedelopez77** [comment]
I tried several runs, with different texts, relaunching the interpreter, and I couldn't reproduce it. I always get the same list of keywords in the exact same order.
Are you using the last version of summa, with the last version of scipy and all its dependencies?

**Comment by vertika-19** [comment]
Are you using python 2?

**Comment by fedelopez77** [comment]
No, the current stable version on Master on this repository, as the one that you can download via pip, are for Python 3. We no longer provide support for Python2

**Comment by vertika-19** [comment]
In the first go I tried few things like I  created 2 separate virtual environments 
1. With python 3
2. With python 2 

Freshly installed just summa on these which installed scipy and numpy as dependencies. 
Installed summa as : pip install summa 

In 1. python 3 - problem persists i.e shows different keywords everytime I launch the interpreter and run 

In 2. Python 2 - No such problem, keywords set and order is same everytime. 

Hopefully it's the same with you.

**Comment by mingoarte** [comment]
Hello! I'm facing the same error with python 3. Somebody solved this?
